src/tree/mod.rs

```diff
@@ -213,7 +213,7 @@ impl RatchetTree {
     }
 
     /// Get a reference to the own key package.
-    fn get_own_key_package_ref(&self) -> &KeyPackage {
+    pub fn get_own_key_package_ref(&self) -> &KeyPackage {
```

Comment:
While you're at it, rename to `own_key_package`?

Reply:
Fixed.